### ISSUE TITLE
Typescript bindings

### ISSUE THREAD

**Issue by bgschiller** [mfogel/polygon-clipping#76]
I'm using this library in a project, and wanted to have TS bindings. Would you be interested in a PR for them? The code is small (it's entirety is below), but I understand that it is a maintenance burden—once you provide the bindings people will expect them to remain up-to-date.

```ts
declare module 'polygon-clipping' {
  export type Pair = [number, number];
  export type Ring = Pair[];
  export type Polygon = Ring[];
  export type MultiPolygon = Polygon[];
  type Geom = Polygon | MultiPolygon;
  export function intersection(geom: Geom, ...geoms: Geom[]): MultiPolygon;
  export function xor(geom: Geom, ...geoms: Geom[]): MultiPolygon;
  export function union(geom: Geom, ...geoms: Geom[]): MultiPolygon;
  export function difference(subjectGeom: Geom, ...clipGeoms: Geom[]): MultiPolygon;
}
```

**Comment by mfogel** [comment]
Yes, I certainly would be interested in a PR! I'd view this project's public API as stable.

**Comment by mfogel** [comment]
Addressed in #77. Thanks for the PR @bgschiller , I plan to release a new minor version to npm this week with this update.

**Comment by ralexrdz** [comment]
I think instead of writing our own types,we should be using geojson types

https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/geojson
